src/client/components/MealsList.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from "react";
+const mealAPI = process.env.REACT_APP_MEALS_URL;
+
+export default function MealList() {
+  const [meals, setMeals] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      const data = await fetch(mealAPI);
+      const res = await data.json();
+      setMeals(res);
+    }
+    console.log(mealAPI);
```

Comment:
A tiny point, it is bettter to remove console.log from code. :)

Reply:
> Perfect, Well done! 👍

Hi @zmzade, 

Thank you so much for your feedback! 
I have edited my varibale and removed console